presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/HiveTypeTranslator.java

```diff
@@ -11,7 +11,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.hive;
+package com.facebook.presto.hive.metastore;
```

Comment:
Why was this file moved to `com.facebook.presto.hive.metastore`? The callsites of the methods in this class are all in `com.facebook.presto.hive`, and it causes circular reference between itself and `HiveType`.

Reply:
Moved it back and implemented a simple translateToHiveType in PartitionFilterBuilder instead